python/cudf/cudf/core/column_accessor.py

```diff
@@ -341,6 +349,80 @@ def _pad_key(self, key: Any, pad_value="") -> Any:
             key = (key,)
         return key + (pad_value,) * (self.nlevels - len(key))
 
+    def replace_level_values(
+        self, mapper: Union[Mapping[Any, Any], Callable], level: Optional[int]
+    ) -> ColumnAccessor:
+        """
+        Parameters
+        ----------
+        self : ColumnAccessor of a given dataframe
+
+        mapper : dict-like or function transformations to apply to
+            the column label values depending on selected ``level``.
+
+            If dict-like, only replace the specified level of the
+            ColumnAccessor's keys (that match the mapper's keys) with
+            mapper's values
+
+            If callable, the function is applied only to the specified level
+            of the ColumnAccessor's keys.
+
+        level : int
+            In case of RangeIndex, only supported level is [0, None].
+            In case of a MultiColumn, only the column labels in the specified
+            level of the ColumnAccessor's keys will be transformed.
+
+        Returns
+        -------
+        A new ColumnAccessor with values in the keys replaced according
+        to the given mapper and level.
+
+        """
+        col_names, columns = list(self.keys()), list(self.values())
+
+        if self.multiindex:
+            if isinstance(mapper, dict):
+                cols_list = [list(col) for col in col_names]
+                new_names = []
+                for col_name in cols_list:
+                    col_name[level] = mapper.get(
```

Comment:
The style check is failing here because `level` is optional, but here it is expected to be an integer. I think we should make `level` non-optional.